pkg/util/ec2/diagnosis.go

```diff
@@ -0,0 +1,21 @@
+package ec2
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/diagnose/diagnosis"
+	log "github.com/cihub/seelog"
+)
+
+func init() {
```

Comment:
in other packages, we put init functions at the end of the files

Reply:
Seems there's no convention on this.
I'd rather have it at the beginning of the file since it's an important part of every package better to see it upfront than scrolling to the bottom of the file to check if there's one.